src/Dismiss.php

```diff
@@ -52,6 +52,7 @@ class Dismiss {
 	 * @since 1.0
 	 * @param string $id     A unique ID for this notice. Can contain lowercase characters and underscores.
 	 * @param string $prefix The prefix that will be used for the option/user-meta.
+	 * @param string $scope
```

Comment:
Thank you for adding these. Could you add a description for this var as well? The param can be one of `user` or `global` and controls where the dismissal will be saved.

Reply:
Done.

Reply:
Thank! I'll go ahead and merge this one :)